.storybook/theme.js

```diff
@@ -0,0 +1,7 @@
+import { create } from "@storybook/theming";
+import logo from "../logo/openmrs_logo.png";
```

Comment:
Could we reuse the logo svg icon for this?

Reply:
to use the svg, it requires the width and height to be set, we haven't set it for the logo svg icons.